experimental/packages/exporter-logs-otlp-grpc/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@opentelemetry/exporter-logs-otlp-grpc",
+  "version": "0.38.0",
+  "description": "OpenTelemetry Collector Exporter allows user to send collected log records to the OpenTelemetry Collector",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js",
+  "scripts": {
+    "prepublishOnly": "npm run compile",
+    "compile": "tsc --build",
+    "clean": "tsc --build --clean",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts'",
+    "version": "node ../../../scripts/version-update.js",
+    "watch": "tsc --watch --build",
+    "precompile": "lerna run version --scope $(npm pkg get name) --include-dependencies",
+    "prewatch": "npm run precompile",
+    "peer-api-check": "node ../../../scripts/peer-api-check.js",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../../"
+  },
+  "keywords": [
+    "opentelemetry",
+    "nodejs",
+    "grpc",
+    "tracing",
+    "profiling",
+    "metrics",
+    "stats",
+    "logging"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=14"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.js.map",
+    "build/src/**/*.d.ts",
+    "build/protos/**/*.proto",
+    "doc",
+    "LICENSE",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@babel/core": "7.16.0",
+    "@grpc/proto-loader": "^0.7.3",
+    "@opentelemetry/api": "1.4.1",
+    "@opentelemetry/api-logs": "0.38.0",
+    "@opentelemetry/otlp-exporter-base": "0.38.0",
+    "@opentelemetry/resources": "1.12.0",
+    "@opentelemetry/sdk-logs": "0.38.0",
```

Comment:
I think per earlier discussions these three SDK packages should be under dependencies?

Reply:
That's the case of `otlp-transformer`. But you are right, this one should be corrected too.